specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -0,0 +1,2280 @@
+        "modelAsString": true
+      }
+    },
+    "OperationResultProperties": {
+      "type": "object",
+      "properties": {},
+      "description": "Operation result properties."
+    },
+    "OperationResultErrorProperties": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "The code of the error.",
+          "readOnly": true
+        },
+        "message": {
+          "type": "string",
+          "description": "The error message.",
+          "readOnly": true
+        }
+      },
+      "description": "Operation result error properties."
+    },
+    "ContactProfileThirdPartyConfiguration": {
+      "type": "object",
+      "required": [
+        "providerName",
+        "missionConfiguration"
+      ],
+      "properties": {
+        "providerName": {
+          "type": "string",
+          "description": "Name of the third-party provider."
+        },
+        "missionConfiguration": {
+          "type": "string",
+          "description": "Name of string referencing the configuration describing contact set-up for a particular mission. Expected values are those which have been created in collaboration with the partner network."
+        }
+      }
+    }
+  },
+  "parameters": {
+    "apiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "enum": [
+        "2020-09-01-preview",
+        "2021-04-04-preview",
+        "2022-03-01",
+        "2022-11-01"
+      ],
+      "x-ms-enum": {
+        "name": "apiVersionParameter",
+        "modelAsString": true
+      },
+      "description": "Version of the API to be used with the client request."
+    },
+    "contactNameParameter": {
+      "name": "contactName",
+      "in": "path",
+      "description": "Contact name.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "contactProfileNameParameter": {
+      "name": "contactProfileName",
+      "in": "path",
+      "description": "Contact Profile name.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "groundStationNameParameter": {
+      "name": "groundStationName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Ground Station name.",
+      "x-ms-parameter-location": "method"
+    },
+    "spacecraftNameParameter": {
+      "name": "spacecraftName",
+      "in": "path",
+      "description": "Spacecraft ID.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "capabilityParameter": {
+      "name": "capability",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "enum": [
+        "EarthObservation",
+        "Communication"
+      ],
+      "x-ms-enum": {
+        "name": "capabilityParameter",
+        "modelAsString": true
+      },
```

Comment:
its recommended to add notations to document any additional constraints enforced on server side such as regex validation, minlength, maxlength